### PR TITLE
Layout/schema popup

### PR DESCRIPTION
**Pull request by sara219**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "react-dom": "^16.10.2",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
-    "styled-components": "^4.4.0"
+    "reactjs-popup": "^1.5.0",
+    "styled-components": "^4.4.0",
+    "styled-react-modal": "^2.0.0"
```
Comment (hshahwan) [inline package.json]: why did you install styled-react-modal?

File: src/Layout/Schema/index.js [hunk 2]
```diff
@@ -1,31 +1,88 @@
...
-    <Container>
-      <Title>Welcome to the GriffinOT Fine Motor Skills programme.</Title>
-      <HorizontalLine />
-      <Para1>
-        Our fine motor skill programme is designed to teach you how to help your child to improve their fine motor skills.
-        <br />
+class Schema extends React.Component {
+
+  state = {
+    Yes: false,
```
Comment (hshahwan) [inline src/Layout/Schema/index.js]: make your variables more readable

File: src/Layout/Schema/index.style.js [hunk 3]
```diff
@@ -49,3 +49,77 @@ export const NextArrow = styled.img`
   width: 3rem;
   margin-left: 1.3rem;
 `;
+
+
+export const BodyPage = styled.html`
```
Comment (hshahwan) [inline src/Layout/Schema/index.style.js]: change the element tag to div
```
Warning: validateDOMNesting(...): <h1> cannot appear as a child of <html>.
````

File: src/Layout/Schema/index.style.js [hunk 4]
```diff
@@ -49,3 +49,77 @@ export const NextArrow = styled.img`
...
+  background: #cfcfcf;
+  justify-content: center;
+`;
+
+export const PopUPCard = styled.body`
```
Comment (hshahwan) [inline src/Layout/Schema/index.style.js]: also here change it to div

File: src/Layout/Schema/index.style.js [hunk 5]
```diff
@@ -49,3 +49,77 @@ export const NextArrow = styled.img`
...
+  display: flex;
+  height: 0.6rem;
+`;
+
+export const PopUpBox = styled.div`
+  width: ${(props) => props.Width || '82rem'};
+  height: ${(props) => props.Height || '100rem'};
+  background: #fffcfc;
+  border: 0.1rem solid #c4c4c4;
+    margin-left: -17.4rem;
```
Comment (hshahwan) [inline src/Layout/Schema/index.style.js]: fix the indentation here.

File: src/Layout/Schema/index.style.js [hunk 6]
```diff
@@ -49,3 +49,77 @@ export const NextArrow = styled.img`
...
+`;
+
+export const ContainerBtn = styled.div`
+  display: flex;
+  justify-content: space-around;
+  margin-top: 15rem;
+`;
+
+
+export const YesBtn = styled.button`
```
Comment (hshahwan) [inline src/Layout/Schema/index.style.js]: why you didn’t create one styled component and add these properties
you repeated them
```  border: 0.3rem #cecece solid;
    background-color: white;
    border-radius: 4rem;
    width: 16rem;
    height: 7rem;
    font-size: 3.5rem;
    outline: none;
```

File: src/Layout/Schema/index.style.js [hunk 7]
```diff
@@ -49,3 +49,77 @@ export const NextArrow = styled.img`
   width: 3rem;
   margin-left: 1.3rem;
 `;
+
```
Comment (hshahwan) [inline src/Layout/Schema/index.style.js:52]: in Component  ```NextText``` how you will pass the on? function(onClick or onchange) and update your state if you created it as paragraph

File: package.json (hunk 1 above)
Comment (sara219) [reply package.json]: @hshahwan  
I removed it

File: src/Layout/Schema/index.js (hunk 2 above)
Comment (sara219) [reply src/Layout/Schema/index.js]: I fix that!

File: src/Layout/Schema/index.style.js (hunk 3 above)
Comment (sara219) [reply src/Layout/Schema/index.style.js]: @hshahwan  
Done!

File: src/Layout/Schema/index.style.js (hunk 4 above)
Comment (sara219) [reply src/Layout/Schema/index.style.js]: @hshahwan
Done!

File: src/Layout/Schema/index.style.js (hunk 5 above)
Comment (sara219) [reply src/Layout/Schema/index.style.js]: @hshahwan
Done!

**Review by someyaaltous — APPROVED** [review]

**Pull request merged**